docs/installation.md

```diff
@@ -5,6 +5,22 @@ title: Installation
 
 You can use Scalafmt from your editor, build tool or terminal.
 
+Create a configuration file called `.scalafmt.conf` at the root of your
+project. As a minimum it should include the Scalafmt version and the
+dialect of Scala you are formatting:
+
+```
+version = @STABLE_VERSION@
+runner.dialect = scala3
```

Comment:
1. move this to the very first example in the configuration page
2. mark each of these two lines there with a `# mandatory` comment
3. use scala213; if we are talking about complete newbies, using scala3 for 2.13 is likely to lead to very unhappy outcomes.